src/YesSql.Core/Commands/CreateDocumentCommand.cs

```diff
@@ -11,18 +11,23 @@ namespace YesSql.Core.Commands
     public class CreateDocumentCommand : DocumentCommand
     {
         private string _tablePrefix;
+        private char _openQuoteDialect;
```

Comment:
This is all over your PR, and that's a good point, but I think we should delegate this to the dialect, and not concatenate them in the sql statements directly.